### ISSUE TITLE
Body type bug in 'Entered Normal Space' event

### ISSUE THREAD

**Issue by Darkcyde13** [EDCD/EDDI#2381]
### Expected
The body type to match that of the game Journal.  In this case, a null body type to equal 'none' in the event.

### Observed
Entering normal space in the vicinity of two stars, resulted in the body type being 'Body' in the event, when it is set as 'null' in the Journal entry.

### Steps to reproduce
1. Enter normal space between a binary pair of stars (but not at a station, etc).  In this case it was stars A & B in the Amenili system.
2. Observe that 'event.bodytype_invariant' is set to 'Body', while the Journal entry is 'Null'.

### Configuration
* Version [Please give full version numbers here, not just "latest" or "current"]
  * EDDI Version: V4.0.1
  * VoiceAttack Version: N/A
* Localization [if applicable]
  * EDDI's language: British English
  * Voice: Ivona Amy

### Investigation Notes
I just jumped into Amenili, and after scanning and mapping, I decided to go visit the Visitor Beacon.  On my way there in supercruise, I was interdicted by a Navy patrol.  On entering normal space, my custom script reported that my ship "has left supercruise near to bodies Amenili AB.", which is not what I was expecting.

After looking over my script and checking the Journal event, I have worked out that EDDI is setting the event.bodytype_invariant variable to be 'Body', when it should be 'None' (or 'null' as that's what the Journal entry has it as).

In the code that checks for 'Planet', 'Star', 'Station', and 'Planetary Ring', I have additional checks for 'Body' (for binary pair planets & moons) and 'None' (for binary pair stars).  This used to work when I made it a long time ago, but I don't often drop between binary pairs, so I've only just noticed this problem now.  Doing a few more 'drops to normal space', I tested the variables (event.bodytype and the invariant version) to find they are both set to be 'Body'.

See below for the Journal entry.  As this states the body type to be 'null', I would expect the EDDI event to use 'None' (or just null) in this situation.  Using 'Body', while _technically_ correct, is not what the game uses.

### EDDI Logs
N/A

### Player journals
Journal event:
`{ "timestamp":"2022-09-04T12:29:37Z", "event":"SupercruiseExit", "Taxi":false, "Multicrew":false, "StarSystem":"Amenili", "SystemAddress":5070611031497, "Body":"Amenili AB", "BodyID":1, "BodyType":"Null" }`

**Comment by Tkael** [comment]
Sure enough, it looks like "Null" is handled like the "None" type... as a generic body.

Looking over my journal history, I _think_ this is only ever written for barycenters so rather than setting this to `None` or `null`, I could set this to `Barycenter` instead (preserving what appears to be unique information that we receive from bodytype "Null"). 

Thoughts?

**Comment by Darkcyde13** [comment]
Your right, using `barycentre` would make much more sense than just null or none.  Assuming there would be no other situations where bodytype would be null, then this would be perfect.

Having moved on now, I'll see if I can find another binary planet pair, and then see what the Journal says when entering normal space between them, just to double check.

Using `barycentre`, I should be able to adapt my code to account for either star or planet pairs. :)

**Comment by Darkcyde13** [comment]
Well, I've tried dropping between several binary pairs of planets, but all I'm getting is that I'm dropping at the nearest of the two bodies, not at a barycentre (so 'Planet', not 'null').  Maybe I do need a target, like the visitor beacon I had targeted before, or maybe a fleet carrier will do too.  Unfortunately, I've taken a trip out to see the 'Stargoid' anomaly, and there isn't any binary pairs with either a beacon or FC in that system to test with.
